filebeat/features/features.go

```diff
@@ -0,0 +1,49 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Elasticsearch B.V. licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package features
+
+import "os"
+
+type void struct{}
+
+// List of input types Elasticsearch state store is enabled for
+var esTypesEnabled = map[string]void{
+	"httpjson": {},
```

Comment:
Can this be configuration instead of in the code, maybe another env var?

Reply:
Sure can do. Something like this? 
`AGENTLESS_ELASTICSEARCH_STATE_STORE_INPUT_TYPES=httpjson,cel`

Reply:
Looks like in order to get the proper docker images posted we need this branch in the main repo. I'll move the branch as is, for testing with agentless team, and them maybe add that env var param.